acquia/mc-cs-plugin-custom-objects
# Maut-10760: fix custom object field values for api request

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [x] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes Maut-10760 



#### Description:

API calls that POST, PUT or PATCH a custom item with changes in a custom field value where not persisting to the database.



#### Steps to test this PR:


1. Create a custom object
2. Test POST, PUT, PATCH, DELETE api methods for custom items.
3. After each API call, do a GET call to verify the results are as expected.
